app/helpers/application_helper.rb

```diff
@@ -12,7 +12,7 @@ def published_document_path(document)
   end
 
   def state(document)
-    state = document.publication_state
+    state = document.publication_state == "live" ? "published" : document.publication_state
```

Comment:
we can call [`document#live?`](https://github.com/alphagov/specialist-publisher-rebuild/blob/master/app/models/document.rb#L54-L58)